com.unity.netcode.gameobjects/Prototyping/NetworkTransform.cs

```diff
@@ -9,7 +9,7 @@ namespace Unity.Netcode.Prototyping
     [AddComponentMenu("Netcode/" + nameof(NetworkTransform))]
     public class NetworkTransform : NetworkBehaviour
     {
-        internal class NetworkState : INetworkSerializable
+        internal struct NetworkState : INetworkSerializable
```

Comment:
I wanted to see what it'd mean to convert from `class` to `struct` (turns out, not too bad :P)